### ISSUE TITLE
New Profile Picture and Banner for Twitter

### ISSUE THREAD

**Issue by DragonflyStats** [cormullion/graphics#22]
Hi there
Can we get a new profile picture and Banner for the JuliaCon 2024 twitter account?

Dimensions : 400px by 400px
The ideal image size for your Twitter profile image is 400px by 400px. Your profile picture will be uploaded as a square, but it's actually displayed as a circle throughout the site. Make sure that your image maintains its integrity when cropped into a circle.

Info on Banner Size here;
https://influencermarketinghub.com/twitter-banner-sizes/

**Comment by cormullion** [comment]
There are some square/round images uploaded - do you want something different?

**Comment by DragonflyStats** [comment]
No. I didnt look properly for those. The twitter account is updated now.
I might keep this issue open for Bluesky / Mastodon for the time being
